### PR TITLE
Create a script to plot total and open issues for current release.

### PR DESCRIPTION
**Pull request by jlewi**
* The new script replaces the existing get_issue_counts.py.
* The corresponding notebooks uses plotly to create an interactive plot.
  The interactive plot is published on plotly and is publicly available.

* Use GitHub's GraphQL API rather than the older REST API

Create a script to identify issues that need attention.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/224)

### CODE REVIEW

File: scripts/project_stats.ipynb [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 194,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<module 'project_stats' from '/home/jlewi/git_kubeflow-community/scripts/project_stats.py'>"
```
Comment (abhi-g) [inline scripts/project_stats.ipynb]: is this intended to run for everyone? If so, might need to replace /home/jlewi/

Comment (jlewi) [reply scripts/project_stats.ipynb]: This is just the output value. So when run on a different machine it would output a different value.

**Pull request merged**